### PR TITLE
 Classification of images using mnist and cifar10

### PR DESCRIPTION
**Pull request by RC2208**
## Related Issue 

- Info about the related issue 

- [ ] LGM-SOC'21 Participant 
- [x] DevIncept Participant
- [x] Contributor

Closes: #464 
### Describe the add-ons or changes you've made
added code for the project.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Compiled and tested on google colab

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done:
Add all the files needed. 
It needs "requirements.txt" file
It needs datasets and should be added in "Dataset" folder.
There should be a separate "README.md" in "Dataset" folder which would add attributes included and dataset link of it.
The project file and README.md file should be in "Model" folder
Also the data visualizations and output screenshots should be in "Images" folder
Add those images links of data visualizations in the README.md file. 
The project title and file names are not in proper structure.
It's better if you add multiple algorithms to work on, that way it would give good efficiency and also you could compare the accuracies and derive a conclusion out of it.
There is a project structure given in README file of the project. You can check it. 

See the project structure [here](https://github.com/prathimacode-hub/ML-ProjectKart/tree/main/Dogecoin%20Price%20Prediction) for reference.

Kindly do all the changes. @RC2208

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
The project title should be as " Classification of Images Using MNIST and CIFAR10".
Add up the "Data Visualizations" you had drawn in "Images" folder and include them as "Data Visualizations" images in README.md file
And add the dataset in "Dataset" folder along with README.md file in it with dataset link and attributes used in this project. 

Kindly do all the changes asked. @RC2208

**Comment by RC2208** [comment]
Ok but the datasets are in the keras library itself so there's no link to it as such. You just import it from keras.datasets so what should I add in the datasets file? @prathimacode-hub

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done:
Yes, change the project title as "Classification of Images using MNIST and CIFAR10".
Then in main README.md, in Dataset section add Dataset link. In Accuracy section, list out the models used along with accuracy. This is bland.
And add the Data Visualization image links you had added in Images README file in the main README file.
If that's the case create a "Dataset" folder, and add "README.md" file in it, then include dataset link of it along with the list of attributes present in the dataset. 

Kindly do these changes. @RC2208

**Review by prathimacode-hub — APPROVED** [review]
Impressive work. Good to see your efforts you had put up. Keep it up. 😎🙌@RC2208
